### DISCUSSION TITLE
Assignment repositories created in GitHub Classroom are not appearing in the students' GitHub accounts

### DISCUSSION THREAD

**Discussion by physcrowley** [community/community#12060] [Other Feature Feedback, Questions, & Ideas] [answered]
This is leading to 403 errors while trying to clone the repos and makes it difficult for students to locate their repos from their GitHub homepage... they have to go into their account overviews to see the recent activity.

This wasn't happening 5 weeks ago. What has changed?

**Comment by physcrowley** [comment]
I have also mentioned this problem in the GitHub Education Community forum where @jsommers originally flagged it about 4 weeks ago in response to a similar query.
https://education.github.community/t/where-are-student-repositories-stored-for-access/72746/7

**Reply by bogdanudrescu-tai** [reply]
Hi @physcrowley ! I have the same problem. Have you found a solution?

**Reply by physcrowley** [reply, marked as answer]
@bogdanudrescu-tai the workaround was to have the students reuse their assignment link to access the repo.

Eventually, after cloning into a local VS Code and creating/syncing a few commits, the repo would appear in their own repo list. They aren't the repo owners, your Classroom's organisation is, so the repo doesn't "live" in their accounts.

tldr; the assignment link always provided reliable access

p.s. I would add that I haven't experienced that 403 error recently, so I don't know if cloning errors are still an issue or if opening the repo in a Codespace would fail either.

**Reply by bogdanudrescu-tai** [reply]
Thank you @physcrowley. Indeed it works. There was a different kind of issue. I am using 2 account to test github classroom and I haven't logout from classroom my teacher account, and while I was using my student account on GitHub, I would still have to logout my teacher account from Classroom. classroom.github.com do not synch automatically the current user from github.com. So I was accepting the task using my teacher account :)
